fire/parser.py

```diff
@@ -94,6 +94,9 @@ def _LiteralEval(value):
     SyntaxError: If the value string has a syntax error.
   """
   root = ast.parse(value, mode='eval')
+  if isinstance(root.body, ast.BinOp):
+    return value
```

Comment:
Let's raise a ValueError here instead, so that the logic of "treat this thing that doesn't parse as a string" all exists in one place (line 74).

Reply:
Ahh right, the fallback logic is to treat it as a string anyway. Good point.